scripts/types/ambient.d.ts

```diff
@@ -19,4 +19,8 @@ declare module "into-stream" {
         export function obj(content: any): NodeJS.ReadableStream
     }
     export = IntoStream;
-}
\ No newline at end of file
+}
+
+declare module "sorcery" {
+    export = any;

```

Comment:
@andy-ms  Ah, I had to install a new nightly build to use `declare module "sorcery";` - for some reason I thought I needed to wait for an LKG. I'll change it.

@mhegazy ...no? (Not that I know of) Is this not supposed to work? Because it does.